### DISCUSSION TITLE
Running without Twitch

### DISCUSSION THREAD

**Discussion by datgamingboi29** [modestimpala/VotVChaosMod#2] [Q&A]
Is there any way to run this mod without twitch and just have random effects happen?

**Comment by modestimpala** [comment]
Not at the moment. It's something I can look into adding. Would it just be a random chance for them to happen? Should it be all effects or only a select few?

**Comment by datgamingboi29** [comment]
Yes. Just a random chance for all effects would be great. If you really want, you could do a setting to add/remove effects from a list

**Comment by modestimpala** [comment]
Hi @datgamingboi29 ,

I've just released version 2.5.0, which includes your requested feature of running without Twitch. It's called Offline Mode and can be enabled through the settings menu. Enjoy!

Moddy
